packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -16,9 +19,7 @@ const propTypes = {
   /**
    * An array of objects to be displayed as quick link options.
    */
-  quickLinks: PropTypes.arrayOf(PropTypes.shape({
-    text: PropTypes.string.isRequired,
-  })),
```

Comment:
it wasn't removed, just changed. I just removed the validation from the nav to leave the more specific validation to the component.